sdk/eventhub/event-hubs/src/index.ts

```diff
@@ -3,10 +3,12 @@
 
 export { EventData, EventHubDeliveryAnnotations, EventHubMessageAnnotations } from "./eventData";
 export { Delivery, AmqpError, Message, MessageHeader, MessageProperties, Dictionary, WebSocketImpl } from "rhea-promise";
-export { ReceiverRuntimeInfo, OnMessage, OnError } from "./eventHubReceiver";
+export { CheckpointData, EventHubReceiver, ReceiverRuntimeInfo, OnMessage, OnError } from "./eventHubReceiver";
+export { ConnectionContext } from "./connectionContext";
 export { ReceiveHandler } from "./streamingReceiver";
+export { LinkEntity, LinkEntityOptions } from "./linkEntity";
 export { EventHubClient, ReceiveOptions, ClientOptionsBase, ClientOptions } from "./eventHubClient";
-export { EventPosition } from "./eventPosition";
+export { EventPosition, EventPositionOptions } from "./eventPosition";
```

Comment:
Can we exclude the fixes to the warnings from the PR? Fixing the warnings would be a good exercise for people working on these projects

Reply:
Yeah, I can pull them back out.